doc/user/content/guides/cdc-mysql.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Change Data Capture (MySQL)"
+description: "How to create real-time materialized views from MySQL changelog data with Materialize."
+weight: 10
+menu:
+  main:
+    parent: guides
+---
+
+Change Data Capture (CDC) allows you to track and propagate changes in a MySQL database to downstream consumers based on its binary log (`binlog`). In this guide, we'll cover how to use Materialize to create and efficiently maintain real-time materialized views on top of CDC data.
+
+## Kafka + Debezium
+
+You can use [Debezium](https://debezium.io/) and the [Kafka source](/sql/create-source/avro-kafka/#debezium-envelope-details) to propagate CDC data from MySQL to Materialize. Debezium captures row-level changes resulting from `INSERT`, `UPDATE` and `DELETE` operations in the upstream database and publishes them as events to Kafka using Kafka Connect-compatible connectors.
+
+### Database Setup
+
+Before deploying a Debezium connector, you need to ensure that the upstream database is configured to support [row-based replication](https://dev.mysql.com/doc/refman/8.0/en/replication-rbr-usage.html). As _root_:
+
+1. Check the `log_bin` and `binlog_format` settings:
+
+    ```sql
+    SHOW VARIABLES
+    WHERE variable_name IN ('log_bin', 'binlog_format');
+    ```
+
+    For CDC, binary logging must be enabled and use the `row` format. If your settings differ, you can adjust the database configuration file (`/etc/mysql/my.cnf`) to use `log_bin=mysql-bin` and `binlog_format=row`. Keep in mind that changing these settings requires a restart of the MySQL instance and can affect database performance.
+
```

Comment:
Can we link to any MySQL doc on the performance implications here?

Reply:
Nice catch, will add it in!